dotnet/src/Functions/Functions.OpenApi.Extensions/Extensions/ApiManifestKernelExtensions.cs

```diff
@@ -87,6 +87,11 @@ public static class ApiManifestKernelExtensions
             var apiDependencyDetails = apiDependency.Value;
 
             var apiDescriptionUrl = apiDependencyDetails.ApiDescriptionUrl;
+            if (apiDescriptionUrl is null)
+            {
+                logger.LogWarning("ApiDescriptionUrl is missing for API dependency: {ApiName}", apiName);
```

Comment:
Apply for the other logger.LogXXX calls touched

```suggestion
                if (logger.IsEnabled(LogLevel.Warning)
                {
                    logger.LogWarning("ApiDescriptionUrl is missing for API dependency: {ApiName}", apiName);
                }
```

Reply:
The called method itself does a check. Other than style, there's not a huge need for such a check. The check is important when there's meaningful work done at the call site which you only want to happen when logging is enabled, eg work done to compute an argument.